src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': completed === true,
+      })}
```

Comment:
let's format it better, if you have 2 function params and one of them spreads to a few lines of code then you should put each of them on a new line. Also, completed is a boolean value - there's no need for comparing it to `true`
```suggestion
      className={classNames(
        'TodoInfo', 
        {
          'TodoInfo--completed': completed 
        }
      )}
```